test/azure/terraform_azure_vm_example_test.go

```diff
@@ -113,6 +113,8 @@ func testInformationOfVM(t *testing.T, terraformOptions *terraform.Options, subs
 	expectedImageSKU := terraform.OutputList(t, terraformOptions, "vm_image_sku")
 	expectedImageVersion := terraform.OutputList(t, terraformOptions, "vm_image_version")
 	expectedAvsName := terraform.Output(t, terraformOptions, "availability_set_name")
+	expectedVMTag02Name := "Environment"
```

Comment:
use terraform output "vm_tags" to get the expected tags values instead

Reply:
Updated to use OutputMap